x/btcstaking/types/keys.go

```diff
@@ -24,7 +24,8 @@ var (
 	// 0x05 was used for something else in the past
 	BTCHeightKey = []byte{0x06} // key prefix for the BTC heights
 	// 0x07 was used for something else in the past
-	PowerDistUpdateKey        = []byte{0x08}             // key prefix for power distribution update events
-	AllowedStakingTxHashesKey = collections.NewPrefix(9) // key prefix for allowed staking tx hashes
-	HeightToVersionMapKey     = []byte{0x10}             // key prefix for height to version map
+	PowerDistUpdateKey        = []byte{0x08}              // key prefix for power distribution update events
+	AllowedStakingTxHashesKey = collections.NewPrefix(9)  // key prefix for allowed staking tx hashes
+	HeightToVersionMapKey     = []byte{0x10}              // key prefix for height to version map
+	LargestBtcReorgInBlocks   = collections.NewPrefix(13) // key prefix for the BTC block height difference of the largest reorg
```

Comment:
key is kept as 13 to avoid collision with main branch keys 
What do you guys think? @babylonlabs-io/core-dev 

```go
        BTCConsumerDelegatorKey   = []byte{0x11}              // key prefix for the Consumer BTC delegators
	BTCStakingEventKey        = []byte{0x12}              // key prefix for the BTC staking events
```

Reply:
Don't think it's necessary as we would have binary upgrade to main anyways

Reply:
addressed to `11`